workflow-runtime/src/commonTest/kotlin/com/squareup/workflow1/internal/WorkflowRunnerTest.kt

```diff
@@ -133,19 +129,21 @@ internal class WorkflowRunnerTest {
     val initialRendering = runner.nextRendering().rendering
     assertEquals("initial props|initial state(initial props)", initialRendering)
 
-    dispatcher.resumeDispatcher()
-
     // The order in which props update and workflow update are processed is deterministic, based
     // on the order they appear in the select block in nextOutput.
-    val firstOutput = scope.async { runner.nextOutput() }
-      .getCompleted()
+    val firstOutput = with(scope.async { runner.nextOutput() }) {
+      scope.advanceUntilIdle()
+      getCompleted()
+    }
```

Comment:
We could throw this in a utility test extension function.

Reply:
+1

There's some subtlety to this, so giving it a function with a good name would probably make it easier to read.

Reply:
Tried this: 268daedd27bf086efe4f39847112db9a7a41810b